oci-test/tests/oci_mysql_configuration_custom/variables.tf

```diff
@@ -0,0 +1,88 @@
+variable "resource_name" {
+  type        = string
+  default     = "turbot-test-20210818-config-custom"
+  description = "Name of the resource used throughout the test."
+}
+
+variable "tenancy_ocid" {
+  type        = string
+  default     = ""
+  description = "OCI tenancy id."
+}
+
+variable "region" {
+  type        = string
+  default     = "ap-mumbai-1"
+  description = "OCI region"
+}
+
+variable "config_file_profile" {
+  type        = string
+  default     = "DEFAULT"
+  description = "OCI credentials profile used for the test. Default is to use the default profile."
+}
+
+variable "oci_ad" {
+  type        = string
+  default     = "TvRS:AP-MUMBAI-1-AD-1"
+  description = "OCI region used for the test. Does not work with default region in config, so must be defined here."
+}
+
+variable "shape" {
+  type        = string
+  default     = "VM.Standard.E2.1"
+  description = "Oracle My SQL Shape."
+}
+
+provider "oci" {
+  tenancy_ocid        = var.tenancy_ocid
+  config_file_profile = var.config_file_profile
+}
+
+locals {
+  path = "${path.cwd}/output.json"
+}
+
+resource "null_resource" "named_test_resource" {
+  provisioner "local-exec" {
+    command = "oci mysql configuration create --compartment-id ${var.tenancy_ocid} --shape-name ${var.shape} --output json > ${local.path}"
```

Comment:
Add Post-test to delete the resource as you are using CLI here, TF will not delete it.